### PR TITLE
rosdep: nixos: Add OpenCV cxxdev output

### PR DESCRIPTION
**Pull request by hacker1024**
Nixpkgs (the Nix package collection) has recently [changed](https://github.com/NixOS/nixpkgs/commit/ada3991349beb5880e3994f25c65a0cf68941b83) the way CUDA dependency resolution works.

Packages that use OpenCV as a C++ dependency should now also include `opencv.cxxdev` along with `opencv` in order to allow CUDA to work correctly. When not using CUDA, this change is harmless.

### CODE REVIEW

**Review by nuclearsandwich — APPROVED** [review]

**Comment by nuclearsandwich** [comment]
rosdep PRs require two approvals before merging.

**Comment by nuclearsandwich** [comment]
@audrow can you provide the second review?

**Review by audrow — APPROVED** [review]

**Pull request merged**
